pkgs/tools/admin/awscli2/default.nix

```
@@ -25,16 +25,22 @@ let
 in
 with py.pkgs; buildPythonApplication rec {
   pname = "awscli2";
-  version = "2.10.1"; # N.B: if you change this, check if overrides are still up-to-date
+  version = "2.10.3"; # N.B: if you change this, check if overrides are still up-to-date
   format = "pyproject";
 
   src = fetchFromGitHub {
     owner = "aws";
     repo = "aws-cli";
     rev = version;
-    hash = "sha256-omg4WNX/DegN6xjHBBGwEijqtvd/b9VfcCC71KfXq98=";
+    hash = "sha256-ogwJTsd2xrWp54utcyG1QO7hGxBC6S4hVlmmGESyPBQ=";
   };
 
+  postPatch = ''
+    substituteInPlace pyproject.toml \
+      --replace "distro>=1.5.0,<1.6.0" "distro>=1.5.0" \
+      --replace "cryptography>=3.3.2,<38.0.5" "cryptography>=39.0.1"
```

Just being pedantic...

```suggestion
      --replace "cryptography>=3.3.2,<38.0.5" "cryptography>=3.3.2"
```

---

I doubt this would make a difference. Because `cryptography` is already at a version beyond that. But. Done. I try not including a version but wouldn't work for whatever reason that I did not bother to dig further. Not worth it.